### DISCUSSION TITLE
Is it possible to customize the expandable icon on DataTable?

### DISCUSSION THREAD

**Discussion by manasirg** [carbon-design-system/carbon-components-svelte#1902] [Q&A]
I would like to display a "Edit" icon instead of "Chevron down". Is there a way to make this switch?

**Comment by metonym** [comment]
Released in [v0.102.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.102.0)

https://svelte.carbondesignsystem.com/components/DataTable#custom-expand-icon
